docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:22.04
+
+MAINTAINER timo.heister@gmail.com
+
+USER root
+
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -yq install \
+    graphviz python3 flex bison make git cmake g++ wget \
+    texlive biber bibtool \
```

Comment:
yes

Reply:
Oh I see now in the `Dockerfile` that you did! Awesome!